webapp/src/browser_routing.js

```diff
@@ -0,0 +1,9 @@
+const WebappUtils = window.WebappUtils;
+
+const navigateToUrl = (urlPath) => {
+    WebappUtils.browserHistory.push(urlPath);
+};
+
+export const navigateToChannel = (team, channel) => {
```

Comment:
Could this be renamed to `navigateToDM`? With the `@` in it, it will only work for DMs.